Pipfile

```diff
@@ -13,11 +13,13 @@ pytest-asyncio = "*"
 asyncmock = "*"
 asyncio = "*"
 python-dotenv = "*"
+py-cord = "*"
```

Comment:
Let's remove Hikari since we don't need it anymore

Reply:
Discord.py replaces Hikari? Isn't Hikari being used in discord_apis?
Also, do I need to pipenv discord myself or should it have already been in this file?

Reply:
@KSBolton yup that's correct. Originally, we were trying the Hikari package and using `discord_apis`, but it didn't work out for us. 
When you run `pipenv install` it will install `discord.py` and the other packages needed to run the project

Reply:
Noted. So will this require rewrite of `discord_apis ` or will all the relevant API calls be handled in `discord_bot.py`?

Reply:
The latter. We can delete `discord_apis` and it's tests file

Reply:
Done.